web/src/components/Spinner/Spinner.js

```diff
@@ -0,0 +1,6 @@
+import 'react-loader-spinner/dist/loader/css/react-spinner-loader.css'
+import Loader from 'react-loader-spinner'
+
+export default () => {
```

Comment:
Instead of adding a new component, import and css, did you consider using:

https://tailwindcss.com/docs/animation#spin